### ISSUE TITLE
Android versiob

### ISSUE THREAD

**Issue by Ridwaan112** [rileytestut/Delta#350]
Please release and android version of the app

**Comment by MisterSheeple** [comment]
I agree, it would be nice. Delta is one of the best emulator front ends I've ever seen. I've reviled being stuck with RetroArch for such a long time because its interface is ugly, so having Delta available on Android would be a win for everyone.

**Comment by Magalhaesmgs** [comment]
Quando sai a versão Android ??

**Comment by Warden20** [comment]
This would be awesome
